plugin/trino-pulsar/src/main/java/io/trino/plugin/pulsar/PulsarTopicDescription.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.pulsar;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import static com.google.common.base.MoreObjects.toStringHelper;
+import static com.google.common.base.Preconditions.checkArgument;
+import static com.google.common.base.Strings.isNullOrEmpty;
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Basic information about a Pulsar topic.
+ */
+public class PulsarTopicDescription
+{
+    private final String tableName;
+    private final String topicName;
+    private final String schemaName;
+
+    @JsonCreator
+    public PulsarTopicDescription(
+            @JsonProperty("tableName") String tableName,
+            @JsonProperty("schemaName") String schemaName,
+            @JsonProperty("topicName") String topicName)
+    {
+        checkArgument(!isNullOrEmpty(tableName), "tableName is null or is empty");
+        this.tableName = tableName;
```

Comment:
Can `tableName` be null? If not please use `requireNonNull` for any of such fields that are non nullable.